configs/configs_mapped/freelancer_mapped/infocard_mapped/infocard/xml.go

```diff
@@ -1,34 +1,44 @@
 package infocard
 
 import (
+	"bytes"
 	"encoding/xml"
+	"io"
 	"strings"
 )
 
-type RDL struct {
-	XMLName xml.Name `xml:"RDL"`
-	TEXT    []string `xml:"TEXT"`
-}
-
 func (i *Infocard) XmlToText() ([]string, error) {
 	return XmlToText(i.content)
 }
 
-func XmlToText(xml_stuff string) ([]string, error) {
-	var structy RDL
-
-	prepared := strings.ReplaceAll(string(xml_stuff), `<?xml version="1.0" encoding="UTF-16"?>`, "")
-
-	err := xml.Unmarshal([]byte(prepared), &structy)
-
-	if _, ok := err.(*xml.SyntaxError); ok {
-		replaced := strings.ReplaceAll(prepared, "&", "")
-		err = xml.Unmarshal([]byte(replaced), &structy)
+func XmlToText(raw string) ([]string, error) {
+	prepared := strings.ReplaceAll(string(raw), `<?xml version="1.0" encoding="UTF-16"?>`, "")
+	decoder := xml.NewDecoder(bytes.NewBufferString(prepared))
+
+	lines := make([]string, 0)
```

Comment:
```suggestion
	lines := make([]string, 0, 10)
```
This one last minor thing to fix and i think it is good to go.
You create by default empty array.
And apperently golang has performance issues with this approach if many times adding minor amount of lines to it in such way.

With adding `10` as third value... you tell Golang to create array with initial... inner memory allocated for 10 elements (but it will be still using it as length of array 0, 1, and etc according what u need)
It optimizes just Memory allocating, saving a lot of performance by wasting a bit more RAM memory.

https://stackoverflow.com/questions/45423667/what-is-the-point-in-setting-a-slices-capacity
in a place like this it can be read
"Golang array capacity" stuff to search.

TLDR: Setting capacity will not change anything, beyond allocating better RAM memory during inner working of array working, and increasing performance twice.